crates/sats/src/hex.rs

```diff
@@ -36,6 +36,18 @@ impl<const N: usize> ops::Deref for HexString<N> {
     }
 }
 
+impl<const N: usize> From<&'_ HexString<N>> for String {
+    fn from(hex: &HexString<N>) -> Self {
+        hex.as_str().into()
+    }
+}
+
+impl<const N: usize> From<HexString<N>> for String {
+    fn from(hex: HexString<N>) -> Self {
+        hex.as_str().into()
+    }
+}
```

Comment:
Why are these impls necessary? Where are they used?

Reply:
Internal tests pass hex strings to these APIs and fail without those impls.